packages/README.md

```diff
@@ -2,23 +2,23 @@
 
 #### Public packages published in the NPM registry
 
-- [@protobuf-ts/plugin](./plugin) - the protoc compiler plugin `protoc-gen-ts`  
+- [@protobuf-ts/plugin](./plugin) - the protoc compiler plugin `protoc-gen-ts`
   [![npm](https://img.shields.io/npm/v/@protobuf-ts/plugin?style=flat-square)](https://www.npmjs.com/package/@protobuf-ts/plugin)
```

Comment:
Whitespace is meaningful in GH flavored markdown. Two newline characters enforce a line break. The change is not relevant here, but might be elsewhere.